modules/dnn/src/ie_ngraph.hpp

```diff
@@ -17,7 +17,11 @@
 #pragma warning(disable : 4245)
 #pragma warning(disable : 4268)
 #endif
+#if INF_ENGINE_VER_MAJOR_GE(INF_ENGINE_RELEASE_2024_0)
```

Comment:
Should be valid even for earlier release: https://github.com/opencv/opencv/blob/0fabe2b9e9fac96957b553b6a83483bddfe129bb/modules/dnn/src/op_inf_engine.hpp#L49.

Not sure about `ov::` namespace